### PR TITLE
docs: 22-restructure-document-following-iso-style

### PR DESCRIPTION
**Pull request by ssteinbuss**
## What this PR changes/adds

Adding mandatory sections as required by ISO style:

* Foreword
* Introduction
* Scope
* normative references
* Terms and definitions

## Why it does that

ISO style is targeted for the document

## Further notes

index.md for respec is updated accordingly. 

## Linked Issue(s)

Closes # <-- _insert issue number if one exists_

_Please be sure to take a look at the [contributing guidelines](../CONTRIBUTING.md#submit-a-pull-request) and our [etiquette for pull requests](../PR_ETIQUETTE.md)._

### CODE REVIEW

**Review by arnoweiss — CHANGES_REQUESTED** [review]
Reach out if there's further explanation needed.

**Comment by jimmarino** [comment]
@ssteinbuss Sorry I accidentally committed @arnoweiss suggestion. I hit the wrong button. Can you revert it?

File: index.html [hunk 1]
```diff
@@ -49,7 +49,16 @@ <h1 id="title">Dataspace Protocol</h1>
...
+<section data-include="foreword.md" data-include-format="markdown">
+</section>
+
+<section data-include="introduction.md" data-include-format="markdown">
+</section>
+
+<section data-include="scope.md" data-include-format="markdown">
+</section>
+
+<section data-include="normative.references.md" data-include-format="markdown">
```
[the remark this answers is not in the document]
Comment (ssteinbuss) [reply index.html:61]: Those are different sections. The normative references section is mandatory in the ISO template and uses a standard text and references to other ISO/IEC standards. The references section in the end should be kept there.

**Review by arnoweiss — APPROVED** [review]
I'll go ahead and fix the review comments in a separate PR. Thanks for creating the new issue.

**Comment by ssteinbuss** [comment]
@jimmarino can you approve please?

**Review by jimmarino — APPROVED** [review]

**Pull request merged**